### ISSUE TITLE
Incorrect wrapper tags with add_object_to_path()

### ISSUE THREAD

**Issue by MScottBlake** [sheagcraig/python-jss#25]
Using this code:
```python
import jss
j = jss.JSS(jss.JSSPrefs())

foo_site = j.Site("Foo")
new_policy = jss.Policy(j, "New Policy TEST")
new_policy.add_object_to_path(foo_site, "general")
```

This is the result:
```xml
<policy>
    <general>
        <name>New Policy TEST</name>
        <enabled>true</enabled>
        ...
         <JSSContainerObject>
            <id>4</id>
            <name>Foo</name>
        </JSSContainerObject>
        ...
    </general>
    ...
</policy>
```

Expected:
```xml
<policy>
    <general>
        <name>New Policy TEST</name>
        <enabled>true</enabled>
        ...
         <site>
            <id>4</id>
            <name>Foo</name>
        </site>
        ...
    </general>
    ...
</policy>
```

**Comment by sheagcraig** [comment]
```jss.JSSObject.add_object_to_path()``` in turn calls ```jss.JSSContainerObject.as_list_data()```, which uses the ```list_type``` property for the tag name to be added.

I didn't add list_type values for all classes because I had no experience using them, so I was hoping exactly this sort of thing would happen, where someone who actually uses it gives me some direction.

I'll add the correct value (it inherits "JSSContainerObject").

That being said; need to refactor out a lot of these properties!

**Comment by MScottBlake** [comment]
This was fixed in #29.
